packages/react-conformance/src/rules.md

```diff
@@ -0,0 +1,77 @@
+# Conformance tests rules
```

Comment:
This probably doesn't need to be checked in (or should be formatted differently) now that we're giving the tests descriptive names.